index.html

```diff
@@ -1259,16 +1257,18 @@ <h3>Compliance</h3>
 
       <p>
         Review any applicable local law when considering developing or
-        operating a decentralized identifier method.
-      </p>
-
-      <p>Consider <a href="https://gdpr.eu/">GDPR</a>.</p>
-
-      <p>Consider <a href="https://oag.ca.gov/privacy/ccpa">CCPA</a>.</p>
-
-      <p>Consider <a
-          href="https://www.bis.doc.gov/index.php/policy-guidance/encryption">EAR</a>.
-      </p>
+        operating a decentralized identifier method. Consider the following:
```

Comment:
```suggestion
        Review any applicable local laws when considering developing or
        operating a decentralized identifier method. Such local laws may 
        include, but are not limited to, the following:
```

Reply:
@TallTed interested in rational for removing links here.

Reply:
Committed: 5f088c6c253d37b0f7635fc81a1ab02c4781c902